file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237
        throw new Error(y, { cause: u });
              ^

Error: [01-May-2025 04:33:31 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>gravity-forms-pdf-extended</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121
[01-May-2025 04:33:44 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>gravity-forms-pdf-extended</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121
[01-May-2025 04:33:55 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>gravity-forms-pdf-extended</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121
[01-May-2025 04:34:04 UTC] PHP Deprecated:  Function get_page_by_title is <strong>deprecated</strong> since version 6.2.0! Use WP_Query instead. in /wordpress/wp-includes/functions.php on line 6121
[01-May-2025 04:34:05 UTC] PHP Fatal error:  Uncaught Error: Undefined constant "Y" in /wordpress/wp-content/plugins/guerrillas-legal-pages/guerrilla-legal-pages.php:286
Stack trace:
#0 /wordpress/wp-includes/class-wp-hook.php(324): guerrilla_legal_terms_pages_install(false)
#1 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#2 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#3 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_guerri...', false)
#4 /internal/eval.php(20): activate_plugin('guerrillas-lega...')
#5 {main}
  thrown in /wordpress/wp-content/plugins/guerrillas-legal-pages/guerrilla-legal-pages.php on line 286
[01-May-2025 04:34:13 UTC] PHP Deprecated:  Function get_page_by_title is <strong>deprecated</strong> since version 6.2.0! Use WP_Query instead. in /wordpress/wp-includes/functions.php on line 6121
[01-May-2025 04:34:14 UTC] PHP Fatal error:  Uncaught Error: Undefined constant "Y" in /wordpress/wp-content/plugins/guerrillas-legal-pages/guerrilla-legal-pages.php:286
Stack trace:
#0 /wordpress/wp-includes/class-wp-hook.php(324): guerrilla_legal_terms_pages_install(false)
#1 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#2 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#3 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_guerri...', false)
#4 /internal/eval.php(20): activate_plugin('guerrillas-lega...')
#5 {main}
  thrown in /wordpress/wp-content/plugins/guerrillas-legal-pages/guerrilla-legal-pages.php on line 286
[01-May-2025 04:34:22 UTC] PHP Deprecated:  Function get_page_by_title is <strong>deprecated</strong> since version 6.2.0! Use WP_Query instead. in /wordpress/wp-includes/functions.php on line 6121
[01-May-2025 04:34:23 UTC] PHP Fatal error:  Uncaught Error: Undefined constant "Y" in /wordpress/wp-content/plugins/guerrillas-legal-pages/guerrilla-legal-pages.php:286
Stack trace:
#0 /wordpress/wp-includes/class-wp-hook.php(324): guerrilla_legal_terms_pages_install(false)
#1 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#2 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#3 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_guerri...', false)
#4 /internal/eval.php(20): activate_plugin('guerrillas-lega...')
#5 {main}
  thrown in /wordpress/wp-content/plugins/guerrillas-legal-pages/guerrilla-legal-pages.php on line 286
[01-May-2025 04:37:02 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function split() in /wordpress/wp-content/plugins/greet-admin-by-time/greetadminbytime.php:31
Stack trace:
#0 /wordpress/wp-includes/class-wp-hook.php(324): change_howdy('Whether theme o...', 'Whether theme o...', 'default')
#1 /wordpress/wp-includes/plugin.php(205): WP_Hook->apply_filters('Whether theme o...', Array)
#2 /wordpress/wp-includes/l10n.php(207): apply_filters('gettext', 'Whether theme o...', 'Whether theme o...', 'default')
#3 /wordpress/wp-includes/l10n.php(307): translate('Whether theme o...', 'default')
#4 /wordpress/wp-includes/theme.php(3941): __('Whether theme o...')
#5 /wordpress/wp-includes/class-wp-hook.php(324): create_initial_theme_features('')
#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#8 /wordpress/wp-settings.php(653): do_action('setup_theme')
#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#12 {main}
  thrown in /wordpress/wp-content/plugins/greet-admin-by-time/greetadminbytime.php on line 31
[01-May-2025 04:37:02 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function split() in /wordpress/wp-content/plugins/greet-admin-by-time/greetadminbytime.php:31
Stack trace:
#0 /wordpress/wp-includes/class-wp-hook.php(324): change_howdy('Error occurred ...', 'Error occurred ...', 'default')
#1 /wordpress/wp-includes/plugin.php(205): WP_Hook->apply_filters('Error occurred ...', Array)
#2 /wordpress/wp-includes/l10n.php(207): apply_filters('gettext', 'Error occurred ...', 'Error occurred ...', 'default')
#3 /wordpress/wp-includes/l10n.php(307): translate('Error occurred ...', 'default')
#4 /wordpress/wp-includes/class-wp-recovery-mode.php(178): __('Error occurred ...')
#5 /wordpress/wp-includes/class-wp-fatal-error-handler.php(55): WP_Recovery_Mode->handle_error(Array)
#6 [internal function]: WP_Fatal_Error_Handler->handle()
#7 {main}
  thrown in /wordpress/wp-content/plugins/greet-admin-by-time/greetadminbytime.php on line 31
[01-May-2025 04:37:10 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function split() in /wordpress/wp-content/plugins/greet-admin-by-time/greetadminbytime.php:31
Stack trace:
#0 /wordpress/wp-includes/class-wp-hook.php(324): change_howdy('Whether theme o...', 'Whether theme o...', 'default')
#1 /wordpress/wp-includes/plugin.php(205): WP_Hook->apply_filters('Whether theme o...', Array)
#2 /wordpress/wp-includes/l10n.php(207): apply_filters('gettext', 'Whether theme o...', 'Whether theme o...', 'default')
#3 /wordpress/wp-includes/l10n.php(307): translate('Whether theme o...', 'default')
#4 /wordpress/wp-includes/theme.php(3941): __('Whether theme o...')
#5 /wordpress/wp-includes/class-wp-hook.php(324): create_initial_theme_features('')
#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#8 /wordpress/wp-settings.php(653): do_action('setup_theme')
#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#12 {main}
  thrown in /wordpress/wp-content/plugins/greet-admin-by-time/greetadminbytime.php on line 31
[01-May-2025 04:37:10 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function split() in /wordpress/wp-content/plugins/greet-admin-by-time/greetadminbytime.php:31
Stack trace:
#0 /wordpress/wp-includes/class-wp-hook.php(324): change_howdy('Error occurred ...', 'Error occurred ...', 'default')
#1 /wordpress/wp-includes/plugin.php(205): WP_Hook->apply_filters('Error occurred ...', Array)
#2 /wordpress/wp-includes/l10n.php(207): apply_filters('gettext', 'Error occurred ...', 'Error occurred ...', 'default')
#3 /wordpress/wp-includes/l10n.php(307): translate('Error occurred ...', 'default')
#4 /wordpress/wp-includes/class-wp-recovery-mode.php(178): __('Error occurred ...')
#5 /wordpress/wp-includes/class-wp-fatal-error-handler.php(55): WP_Recovery_Mode->handle_error(Array)
#6 [internal function]: WP_Fatal_Error_Handler->handle()
#7 {main}
  thrown in /wordpress/wp-content/plugins/greet-admin-by-time/greetadminbytime.php on line 31
[01-May-2025 04:37:19 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function split() in /wordpress/wp-content/plugins/greet-admin-by-time/greetadminbytime.php:31
Stack trace:
#0 /wordpress/wp-includes/class-wp-hook.php(324): change_howdy('Whether theme o...', 'Whether theme o...', 'default')
#1 /wordpress/wp-includes/plugin.php(205): WP_Hook->apply_filters('Whether theme o...', Array)
#2 /wordpress/wp-includes/l10n.php(207): apply_filters('gettext', 'Whether theme o...', 'Whether theme o...', 'default')
#3 /wordpress/wp-includes/l10n.php(307): translate('Whether theme o...', 'default')
#4 /wordpress/wp-includes/theme.php(3941): __('Whether theme o...')
#5 /wordpress/wp-includes/class-wp-hook.php(324): create_initial_theme_features('')
#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#8 /wordpress/wp-settings.php(653): do_action('setup_theme')
#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#12 {main}
  thrown in /wordpress/wp-content/plugins/greet-admin-by-time/greetadminbytime.php on line 31
[01-May-2025 04:37:19 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function split() in /wordpress/wp-content/plugins/greet-admin-by-time/greetadminbytime.php:31
Stack trace:
#0 /wordpress/wp-includes/class-wp-hook.php(324): change_howdy('Error occurred ...', 'Error occurred ...', 'default')
#1 /wordpress/wp-includes/plugin.php(205): WP_Hook->apply_filters('Error occurred ...', Array)
#2 /wordpress/wp-includes/l10n.php(207): apply_filters('gettext', 'Error occurred ...', 'Error occurred ...', 'default')
#3 /wordpress/wp-includes/l10n.php(307): translate('Error occurred ...', 'default')
#4 /wordpress/wp-includes/class-wp-recovery-mode.php(178): __('Error occurred ...')
#5 /wordpress/wp-includes/class-wp-fatal-error-handler.php(55): WP_Recovery_Mode->handle_error(Array)
#6 [internal function]: WP_Fatal_Error_Handler->handle()
#7 {main}
  thrown in /wordpress/wp-content/plugins/greet-admin-by-time/greetadminbytime.php on line 31
[01-May-2025 04:40:17 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>woocommerce</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121
[01-May-2025 04:40:22 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'activation', '2025-05-01 06:40:22')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT 1 FROM sqlite_temp_schema WHERE type = 'table' AND name = ? | parameters: wp_f24_installation_log</li>
<li>Executing: 
				SELECT column_name, is_nullable, column_default, data_type, extra
				FROM _wp_sqlite_mysql_information_schema_columns
				WHERE table_schema = ?
				AND table_name = ?
				ORDER BY ordinal_position
			 | parameters: database_name_here, wp_f24_installation_log</li>
<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) SELECT `column1`, `column2`, `column3`, `column4` FROM (VALUES ( '' , '8.0.22' , 'activation' , '2025-05-01 06:40:22' )) WHERE true | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 20 datatype mismatch
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(690): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 0, Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(472): WP_SQLite_Driver->query('INSERT INTO `wp...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(413): WP_SQLite_DB->_do_query('INSERT INTO `wp...')
#3 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#4 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')
#5 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)
#6 /wordpress/wp-content/plugins/fattura24/fattura24.php(312): Fattura24\fatt_24_insert_installation_log('activation')
#7 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#8 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#9 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#10 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#11 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#12 {main}</pre>
 for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'activation', '2025-05-01 06:40:22') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:40:22 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated billing cb postmeta', '2025-05-01 06:40:22')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT 1 FROM sqlite_temp_schema WHERE type = 'table' AND name = ? | parameters: wp_f24_installation_log</li>
<li>Executing: 
				SELECT column_name, is_nullable, column_default, data_type, extra
				FROM _wp_sqlite_mysql_information_schema_columns
				WHERE table_schema = ?
				AND table_name = ?
				ORDER BY ordinal_position
			 | parameters: database_name_here, wp_f24_installation_log</li>
<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) SELECT `column1`, `column2`, `column3`, `column4` FROM (VALUES ( '' , '8.0.22' , 'updated billing cb postmeta' , '2025-05-01 06:40:22' )) WHERE true | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 20 datatype mismatch
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(690): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 0, Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(472): WP_SQLite_Driver->query('INSERT INTO `wp...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(413): WP_SQLite_DB->_do_query('INSERT INTO `wp...')
#3 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#4 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')
#5 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)
#6 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\fatt_24_insert_installation_log('updated billing...')
#7 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\fatt_24_update_actions()
#8 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#9 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#10 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#11 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#12 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#13 {main}</pre>
 for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated billing cb postmeta', '2025-05-01 06:40:22') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_actions, Fattura24\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:40:22 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated old wc orders postmeta', '2025-05-01 06:40:22')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT 1 FROM sqlite_temp_schema WHERE type = 'table' AND name = ? | parameters: wp_f24_installation_log</li>
<li>Executing: 
				SELECT column_name, is_nullable, column_default, data_type, extra
				FROM _wp_sqlite_mysql_information_schema_columns
				WHERE table_schema = ?
				AND table_name = ?
				ORDER BY ordinal_position
			 | parameters: database_name_here, wp_f24_installation_log</li>
<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) SELECT `column1`, `column2`, `column3`, `column4` FROM (VALUES ( '' , '8.0.22' , 'updated old wc orders postmeta' , '2025-05-01 06:40:22' )) WHERE true | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 20 datatype mismatch
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(690): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 0, Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(472): WP_SQLite_Driver->query('INSERT INTO `wp...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(413): WP_SQLite_DB->_do_query('INSERT INTO `wp...')
#3 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#4 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')
#5 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)
#6 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\fatt_24_insert_installation_log('updated old wc ...')
#7 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\fatt_24_update_actions()
#8 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#9 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#10 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#11 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#12 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#13 {main}</pre>
 for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated old wc orders postmeta', '2025-05-01 06:40:22') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_actions, Fattura24\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:40:22 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated order creation options', '2025-05-01 06:40:22')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT 1 FROM sqlite_temp_schema WHERE type = 'table' AND name = ? | parameters: wp_f24_installation_log</li>
<li>Executing: 
				SELECT column_name, is_nullable, column_default, data_type, extra
				FROM _wp_sqlite_mysql_information_schema_columns
				WHERE table_schema = ?
				AND table_name = ?
				ORDER BY ordinal_position
			 | parameters: database_name_here, wp_f24_installation_log</li>
<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) SELECT `column1`, `column2`, `column3`, `column4` FROM (VALUES ( '' , '8.0.22' , 'updated order creation options' , '2025-05-01 06:40:22' )) WHERE true | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 20 datatype mismatch
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(690): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 0, Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(472): WP_SQLite_Driver->query('INSERT INTO `wp...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(413): WP_SQLite_DB->_do_query('INSERT INTO `wp...')
#3 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#4 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')
#5 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)
#6 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\fatt_24_insert_installation_log('updated order c...')
#7 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\fatt_24_update_actions()
#8 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#9 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#10 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#11 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#12 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#13 {main}</pre>
 for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated order creation options', '2025-05-01 06:40:22') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_actions, Fattura24\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:40:22 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax settings', '2025-05-01 06:40:22')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT 1 FROM sqlite_temp_schema WHERE type = 'table' AND name = ? | parameters: wp_f24_installation_log</li>
<li>Executing: 
				SELECT column_name, is_nullable, column_default, data_type, extra
				FROM _wp_sqlite_mysql_information_schema_columns
				WHERE table_schema = ?
				AND table_name = ?
				ORDER BY ordinal_position
			 | parameters: database_name_here, wp_f24_installation_log</li>
<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) SELECT `column1`, `column2`, `column3`, `column4` FROM (VALUES ( '' , '8.0.22' , 'updated fattura24 tax settings' , '2025-05-01 06:40:22' )) WHERE true | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 20 datatype mismatch
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(690): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 0, Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(472): WP_SQLite_Driver->query('INSERT INTO `wp...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(413): WP_SQLite_DB->_do_query('INSERT INTO `wp...')
#3 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#4 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')
#5 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)
#6 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\fatt_24_insert_installation_log('updated fattura...')
#7 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\fatt_24_update_actions()
#8 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#9 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#10 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#11 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#12 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#13 {main}</pre>
 for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax settings', '2025-05-01 06:40:22') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_actions, Fattura24\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:40:22 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax table', '2025-05-01 06:40:22')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT 1 FROM sqlite_temp_schema WHERE type = 'table' AND name = ? | parameters: wp_f24_installation_log</li>
<li>Executing: 
				SELECT column_name, is_nullable, column_default, data_type, extra
				FROM _wp_sqlite_mysql_information_schema_columns
				WHERE table_schema = ?
				AND table_name = ?
				ORDER BY ordinal_position
			 | parameters: database_name_here, wp_f24_installation_log</li>
<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) SELECT `column1`, `column2`, `column3`, `column4` FROM (VALUES ( '' , '8.0.22' , 'updated fattura24 tax table' , '2025-05-01 06:40:22' )) WHERE true | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 20 datatype mismatch
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(690): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 0, Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(472): WP_SQLite_Driver->query('INSERT INTO `wp...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(413): WP_SQLite_DB->_do_query('INSERT INTO `wp...')
#3 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#4 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')
#5 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)
#6 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\fatt_24_insert_installation_log('updated fattura...')
#7 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\fatt_24_update_actions()
#8 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#9 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#10 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#11 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#12 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#13 {main}</pre>
 for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax table', '2025-05-01 06:40:22') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_actions, Fattura24\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:40:22 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated invoice creation options', '2025-05-01 06:40:22')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT 1 FROM sqlite_temp_schema WHERE type = 'table' AND name = ? | parameters: wp_f24_installation_log</li>
<li>Executing: 
				SELECT column_name, is_nullable, column_default, data_type, extra
				FROM _wp_sqlite_mysql_information_schema_columns
				WHERE table_schema = ?
				AND table_name = ?
				ORDER BY ordinal_position
			 | parameters: database_name_here, wp_f24_installation_log</li>
<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) SELECT `column1`, `column2`, `column3`, `column4` FROM (VALUES ( '' , '8.0.22' , 'updated invoice creation options' , '2025-05-01 06:40:22' )) WHERE true | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 20 datatype mismatch
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(690): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 0, Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(472): WP_SQLite_Driver->query('INSERT INTO `wp...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(413): WP_SQLite_DB->_do_query('INSERT INTO `wp...')
#3 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#4 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')
#5 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)
#6 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\fatt_24_insert_installation_log('updated invoice...')
#7 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\fatt_24_update_actions()
#8 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#9 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#10 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#11 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#12 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#13 {main}</pre>
 for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated invoice creation options', '2025-05-01 06:40:22') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_actions, Fattura24\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:40:23 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fe issue number default option', '2025-05-01 06:40:23')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT 1 FROM sqlite_temp_schema WHERE type = 'table' AND name = ? | parameters: wp_f24_installation_log</li>
<li>Executing: 
				SELECT column_name, is_nullable, column_default, data_type, extra
				FROM _wp_sqlite_mysql_information_schema_columns
				WHERE table_schema = ?
				AND table_name = ?
				ORDER BY ordinal_position
			 | parameters: database_name_here, wp_f24_installation_log</li>
<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) SELECT `column1`, `column2`, `column3`, `column4` FROM (VALUES ( '' , '8.0.22' , 'updated fe issue number default option' , '2025-05-01 06:40:23' )) WHERE true | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 20 datatype mismatch
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(690): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 0, Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(472): WP_SQLite_Driver->query('INSERT INTO `wp...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(413): WP_SQLite_DB->_do_query('INSERT INTO `wp...')
#3 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#4 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')
#5 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)
#6 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\fatt_24_insert_installation_log('updated fe issu...')
#7 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\fatt_24_update_actions()
#8 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#9 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#10 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#11 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#12 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#13 {main}</pre>
 for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fe issue number default option', '2025-05-01 06:40:23') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_actions, Fattura24\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:41:01 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>woocommerce</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121
[01-May-2025 04:41:06 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>DESC wp_fattura_tax</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown query type: DESC.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('DESC wp_fattura...')
#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(43): wpdb->get_col('DESC wp_fattura...', 0)
#3 /wordpress/wp-content/plugins/fattura24/fattura24.php(309): Fattura24\fatt_24_update_tax_table()
#4 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#8 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#9 {main}</pre>
 for query DESC wp_fattura_tax made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:41:06 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)
				VALUES (:table, :column, :datatype)
				ON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype
			 | parameters: wp_fattura_tax, BLOG_ID, int(1)</li>
<li>Executing: ALTER TABLE wp_fattura_tax ADD blog_id integer   | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: blog_id.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/fattura24/src/uty.php(46): WP_SQLite_DB->query('ALTER TABLE wp_...')
#2 /wordpress/wp-content/plugins/fattura24/fattura24.php(309): Fattura24\fatt_24_update_tax_table()
#3 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#7 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#8 {main}</pre>
 for query ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:41:06 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>DESC wp_fattura_tax</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown query type: DESC.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('DESC wp_fattura...')
#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(27): wpdb->get_col('DESC wp_fattura...', 0)
#3 /wordpress/wp-content/plugins/fattura24/fattura24.php(311): Fattura24\fatt_24_add_column_to_tax_table('used_for_shippi...', 'INT', 1, 0)
#4 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#8 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#9 {main}</pre>
 for query DESC wp_fattura_tax made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_add_column_to_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:41:06 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'activation', '2025-05-01 06:41:06')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, activation, 2025-05-01 06:41:06</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)
#4 /wordpress/wp-content/plugins/fattura24/fattura24.php(312): Fattura24\fatt_24_insert_installation_log('activation')
#5 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#9 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#10 {main}</pre>
 for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'activation', '2025-05-01 06:41:06') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:41:06 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated billing cb postmeta', '2025-05-01 06:41:06')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated billing cb postmeta, 2025-05-01 06:41:06</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)
#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\fatt_24_insert_installation_log('updated billing...')
#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\fatt_24_update_actions()
#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#11 {main}</pre>
 for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated billing cb postmeta', '2025-05-01 06:41:06') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_actions, Fattura24\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:41:06 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated old wc orders postmeta', '2025-05-01 06:41:06')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated old wc orders postmeta, 2025-05-01 06:41:06</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)
#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\fatt_24_insert_installation_log('updated old wc ...')
#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\fatt_24_update_actions()
#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#11 {main}</pre>
 for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated old wc orders postmeta', '2025-05-01 06:41:06') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_actions, Fattura24\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:41:06 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated order creation options', '2025-05-01 06:41:06')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated order creation options, 2025-05-01 06:41:06</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)
#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\fatt_24_insert_installation_log('updated order c...')
#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\fatt_24_update_actions()
#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#11 {main}</pre>
 for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated order creation options', '2025-05-01 06:41:06') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_actions, Fattura24\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:41:06 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax settings', '2025-05-01 06:41:06')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated fattura24 tax settings, 2025-05-01 06:41:06</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)
#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\fatt_24_insert_installation_log('updated fattura...')
#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\fatt_24_update_actions()
#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#11 {main}</pre>
 for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax settings', '2025-05-01 06:41:06') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_actions, Fattura24\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:41:06 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>DESC wp_fattura_tax</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown query type: DESC.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('DESC wp_fattura...')
#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(43): wpdb->get_col('DESC wp_fattura...', 0)
#3 /wordpress/wp-content/plugins/fattura24/src/uty.php(80): Fattura24\fatt_24_update_tax_table()
#4 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\fatt_24_update_actions()
#5 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#9 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#10 {main}</pre>
 for query DESC wp_fattura_tax made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_actions, Fattura24\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:41:06 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)
				VALUES (:table, :column, :datatype)
				ON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype
			 | parameters: wp_fattura_tax, BLOG_ID, int(1)</li>
<li>Executing: ALTER TABLE wp_fattura_tax ADD blog_id integer   | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: blog_id.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/fattura24/src/uty.php(46): WP_SQLite_DB->query('ALTER TABLE wp_...')
#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(80): Fattura24\fatt_24_update_tax_table()
#3 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\fatt_24_update_actions()
#4 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#8 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#9 {main}</pre>
 for query ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_actions, Fattura24\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:41:06 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax table', '2025-05-01 06:41:06')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated fattura24 tax table, 2025-05-01 06:41:06</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)
#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\fatt_24_insert_installation_log('updated fattura...')
#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\fatt_24_update_actions()
#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#11 {main}</pre>
 for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax table', '2025-05-01 06:41:06') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_actions, Fattura24\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:41:06 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated invoice creation options', '2025-05-01 06:41:06')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated invoice creation options, 2025-05-01 06:41:06</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)
#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\fatt_24_insert_installation_log('updated invoice...')
#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\fatt_24_update_actions()
#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#11 {main}</pre>
 for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated invoice creation options', '2025-05-01 06:41:06') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_actions, Fattura24\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:41:07 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fe issue number default option', '2025-05-01 06:41:07')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated fe issue number default option, 2025-05-01 06:41:07</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)
#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\fatt_24_insert_installation_log('updated fe issu...')
#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\fatt_24_update_actions()
#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#11 {main}</pre>
 for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fe issue number default option', '2025-05-01 06:41:07') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_actions, Fattura24\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:41:44 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>woocommerce</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121
[01-May-2025 04:41:48 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>DESC wp_fattura_tax</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown query type: DESC.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('DESC wp_fattura...')
#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(43): wpdb->get_col('DESC wp_fattura...', 0)
#3 /wordpress/wp-content/plugins/fattura24/fattura24.php(309): Fattura24\fatt_24_update_tax_table()
#4 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#8 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#9 {main}</pre>
 for query DESC wp_fattura_tax made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:41:48 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)
				VALUES (:table, :column, :datatype)
				ON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype
			 | parameters: wp_fattura_tax, BLOG_ID, int(1)</li>
<li>Executing: ALTER TABLE wp_fattura_tax ADD blog_id integer   | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: blog_id.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/fattura24/src/uty.php(46): WP_SQLite_DB->query('ALTER TABLE wp_...')
#2 /wordpress/wp-content/plugins/fattura24/fattura24.php(309): Fattura24\fatt_24_update_tax_table()
#3 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#7 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#8 {main}</pre>
 for query ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:41:48 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>DESC wp_fattura_tax</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown query type: DESC.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('DESC wp_fattura...')
#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(27): wpdb->get_col('DESC wp_fattura...', 0)
#3 /wordpress/wp-content/plugins/fattura24/fattura24.php(311): Fattura24\fatt_24_add_column_to_tax_table('used_for_shippi...', 'INT', 1, 0)
#4 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#8 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#9 {main}</pre>
 for query DESC wp_fattura_tax made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_add_column_to_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:41:48 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'activation', '2025-05-01 06:41:48')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, activation, 2025-05-01 06:41:48</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)
#4 /wordpress/wp-content/plugins/fattura24/fattura24.php(312): Fattura24\fatt_24_insert_installation_log('activation')
#5 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#9 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#10 {main}</pre>
 for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'activation', '2025-05-01 06:41:48') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:41:48 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated billing cb postmeta', '2025-05-01 06:41:48')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated billing cb postmeta, 2025-05-01 06:41:48</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)
#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\fatt_24_insert_installation_log('updated billing...')
#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\fatt_24_update_actions()
#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#11 {main}</pre>
 for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated billing cb postmeta', '2025-05-01 06:41:48') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_actions, Fattura24\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:41:48 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated old wc orders postmeta', '2025-05-01 06:41:48')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated old wc orders postmeta, 2025-05-01 06:41:48</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)
#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\fatt_24_insert_installation_log('updated old wc ...')
#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\fatt_24_update_actions()
#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#11 {main}</pre>
 for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated old wc orders postmeta', '2025-05-01 06:41:48') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_actions, Fattura24\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:41:48 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated order creation options', '2025-05-01 06:41:48')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated order creation options, 2025-05-01 06:41:48</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)
#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\fatt_24_insert_installation_log('updated order c...')
#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\fatt_24_update_actions()
#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#11 {main}</pre>
 for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated order creation options', '2025-05-01 06:41:48') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_actions, Fattura24\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:41:48 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax settings', '2025-05-01 06:41:48')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated fattura24 tax settings, 2025-05-01 06:41:48</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)
#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\fatt_24_insert_installation_log('updated fattura...')
#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\fatt_24_update_actions()
#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#11 {main}</pre>
 for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax settings', '2025-05-01 06:41:48') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_actions, Fattura24\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:41:48 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>DESC wp_fattura_tax</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown query type: DESC.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('DESC wp_fattura...')
#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(43): wpdb->get_col('DESC wp_fattura...', 0)
#3 /wordpress/wp-content/plugins/fattura24/src/uty.php(80): Fattura24\fatt_24_update_tax_table()
#4 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\fatt_24_update_actions()
#5 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#8 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#9 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#10 {main}</pre>
 for query DESC wp_fattura_tax made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_actions, Fattura24\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:41:48 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)
				VALUES (:table, :column, :datatype)
				ON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype
			 | parameters: wp_fattura_tax, BLOG_ID, int(1)</li>
<li>Executing: ALTER TABLE wp_fattura_tax ADD blog_id integer   | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: blog_id.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/fattura24/src/uty.php(46): WP_SQLite_DB->query('ALTER TABLE wp_...')
#2 /wordpress/wp-content/plugins/fattura24/src/uty.php(80): Fattura24\fatt_24_update_tax_table()
#3 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\fatt_24_update_actions()
#4 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#8 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#9 {main}</pre>
 for query ALTER TABLE wp_fattura_tax ADD blog_id INT(1) AFTER tax_code made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_actions, Fattura24\fatt_24_update_tax_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:41:48 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax table', '2025-05-01 06:41:48')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated fattura24 tax table, 2025-05-01 06:41:48</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)
#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\fatt_24_insert_installation_log('updated fattura...')
#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\fatt_24_update_actions()
#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#11 {main}</pre>
 for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fattura24 tax table', '2025-05-01 06:41:48') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_actions, Fattura24\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:41:48 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated invoice creation options', '2025-05-01 06:41:48')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated invoice creation options, 2025-05-01 06:41:48</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)
#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\fatt_24_insert_installation_log('updated invoice...')
#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\fatt_24_update_actions()
#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#11 {main}</pre>
 for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated invoice creation options', '2025-05-01 06:41:48') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_actions, Fattura24\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:41:49 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fe issue number default option', '2025-05-01 06:41:49')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES (:param0 , :param1 , :param2 , :param3 ) | parameters: , 8.0.22, updated fe issue number default option, 2025-05-01 06:41:49</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 20 datatype mismatch.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_f24_installa...', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/fattura24/src/behaviour.php(303): wpdb->insert('wp_f24_installa...', Array)
#4 /wordpress/wp-content/plugins/fattura24/src/uty.php(96): Fattura24\fatt_24_insert_installation_log('updated fe issu...')
#5 /wordpress/wp-content/plugins/fattura24/fattura24.php(315): Fattura24\fatt_24_update_actions()
#6 /wordpress/wp-includes/class-wp-hook.php(324): Fattura24\{closure}(false)
#7 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#8 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#9 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_fattur...', false)
#10 /internal/eval.php(20): activate_plugin('fattura24/fattu...')
#11 {main}</pre>
 for query INSERT INTO `wp_f24_installation_log` (`id`, `vers`, `event_type`, `event_date`) VALUES ('', '8.0.22', 'updated fe issue number default option', '2025-05-01 06:41:49') made by activate_plugin, do_action('activate_fattura24/fattura24.php'), WP_Hook->do_action, WP_Hook->apply_filters, Fattura24\{closure}, Fattura24\fatt_24_update_actions, Fattura24\fatt_24_insert_installation_log, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[01-May-2025 04:44:33 UTC] PHP Fatal error:  Uncaught Error: Class 'GFForms' not found in /wordpress/wp-content/plugins/gf-loqate-bank-verification/src/Plugin.php:20
Stack trace:
#0 /wordpress/wp-content/plugins/gf-loqate-bank-verification/gf-loqate-bank-verification.php(30): Itineris\GFLoqateBankVerification\Plugin::run()
#1 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#2 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('gf-loqate-bank-...')
#3 /internal/eval.php(20): activate_plugin('gf-loqate-bank-...')
#4 {main}
  thrown in /wordpress/wp-content/plugins/gf-loqate-bank-verification/src/Plugin.php on line 20
[01-May-2025 04:44:41 UTC] PHP Fatal error:  Uncaught Error: Class 'GFForms' not found in /wordpress/wp-content/plugins/gf-loqate-bank-verification/src/Plugin.php:20
Stack trace:
#0 /wordpress/wp-content/plugins/gf-loqate-bank-verification/gf-loqate-bank-verification.php(30): Itineris\GFLoqateBankVerification\Plugin::run()
#1 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#2 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('gf-loqate-bank-...')
#3 /internal/eval.php(20): activate_plugin('gf-loqate-bank-...')
#4 {main}
  thrown in /wordpress/wp-content/plugins/gf-loqate-bank-verification/src/Plugin.php on line 20

    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
  [cause]: Error: Error when executing the blueprint step #1 ({"step":"installPlugin","pluginData":{"resource":"wordpress.org/plugins","slug":"gf-loqate-bank-verification"},"options":{"activate":true}}) : PHP.run() failed with exit code 255 and the following output: 
      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)
      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)
      ... 2 lines matching cause stack trace ...
      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
    [cause]: PHPExecutionFailureError: PHP.run() failed with exit code 255 and the following output: 
        at PHP.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/universal/index.js:987:19)
        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
        at async wr (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:33:39)
        at async Object.gt [as installPlugin] (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:974:42)
        at async g (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12517:30)
        at async Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12446:23)
        at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)
        at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)
        at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
        at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
      response: PHPResponse {
        httpStatusCode: 500,
        headers: {
          'x-powered-by': [ 'PHP/7.2.34-dev' ],
          'content-type': [ 'text/html; charset=UTF-8' ]
        },
        bytes: Uint8Array(0) [],
        exitCode: 255,
        errors: ''
      },
      source: 'request'
    }
  }
}

Node.js v23.11.0
